### DISCUSSION TITLE
Application running inside an iFrame that should not show any login form

### DISCUSSION THREAD

**Discussion by xennialex** [nextauthjs/next-auth#4235] [Help]
Hi,

I have an NEXTjs application with next-auth that is running inside an iFrame. The "main" application will only show this IFrame if the user is already authenticated with an Shibboleth authentication server.

Before we started optimizing it for the iFrame - which we MUST use - we added an custom OIDC provider and were redirected to the login page and then back to our application. 

Is there a tutorial or documentation how to use next-auth so it only gets the access token from the authentication server but will never try to login itself?

**Comment by gurhangulmez** [comment]
Any solution so far?
